ti/savings/savings.go

```diff
@@ -44,7 +46,22 @@ func ParseAndUploadSavings(ctx context.Context, workspace string, log *logrus.Lo
 		}
 	}
 
-	// DLC Savings (Placeholder)
+	// DLC Savings
+	if cacheMetricsFile, found := envs["PLUGIN_CACHE_METRICS_FILE"]; found {
+		if opts, ok := envs["PLUGIN_BUILDER_DRIVER_OPTS"]; ok && strings.Contains(opts, "harness/buildkit") {
+			dlcState, err := dlc.GetFeatureState(cacheMetricsFile, log)
+			if err == nil {
+				states = append(states, dlcState)
+				log.Infof("Computed docker layer caching execution details with state %s and time %dms", dlcState, cmdTimeTaken)
+				tiStart := time.Now()
+				tiErr := tiConfig.GetClient().WriteSavings(ctx, stepID, types.DLC, dlcState, cmdTimeTaken, types.SavingsRequest{})
```

Comment:
If you're already parsing the data, it doesn't hurt to send the savings details as well. We can add the server side changes later for DLC

Reply:
Will do it later once we have a need for it